src/main/features/response/views/reject-all-of-claim.njk

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t("Money claim") %}
+
+{% set heading = t("I reject all of the claim") %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        {% set error = form.errorFor('type') %}
+        <div class="form-group {% if error %} form-group-error {% endif %}">
+          <fieldset id="type[label]">
+            <legend class="visually-hidden">{{ t("I reject all of the claim") }}</legend>
+            {% if error %}<span class="error-message">{{ t(error) }}</span>{% endif %}
+        {{
+        radioGroup(
+        label = "Why do you reject all the claim?",
+        name = "option",
+        options = [
+        {
+          label: "I've already paid the claim in full",
+          value: "paid",
+          hint: "",
+          bold: true
+        },
+        {
+          label: "I dispute the claim",
+          value: "dispute",
+          hint: "",
+          bold: true
+        },
+        {
+          label: 'Defend now and also make a counterclaim',
+          value: "counterClaim",
```

Comment:
I do not see this being used anywhere in the application. How does is fit into the counter claim feature we already have? Either start using it or delete.